view/rooms/app.js

```diff
@@ -258,6 +265,13 @@ function setupSocketIOClient(bots) {
           detail: events,
         }));
       });
+
+      if (events.context.type === 'RoomState') {
```

Comment:
check if `events.context` exist before extracting `type` value to prevent `Can't ready property type of undefined` errors